src/components/table/examples/birds.tsx

```diff
@@ -155,7 +155,7 @@ export const data: Bird[] = [
         origin: ['Europe', 'Asia'],
     },
     {
-        name: "Wilson's Snipe",
+        name: "Wilson's Snipe", // eslint-disable-line quotes
```

Comment:
Isn't there a setting that automatically allows double quotes if it helps avoid escaping single quotes in the string?

Reply:
Yeah maybe :thinking: 
I think there were two rules conflicting each other, so maybe one of them should just be disabled? :thinking:

Reply:
:zap: